### ISSUE TITLE
CORS trouble in Chrome and Safari

### ISSUE THREAD

**Issue by arggh** [exupero/saveSvgAsPng#126]
I have my images in an Amazon S3 bucket that's being served via CloudFront.

I have configured my CloudFront cache behavior so that the `Origin` header is whitelisted and included. With Firefox, using `saveSvgAsPng` works as expected and the images appear nice and dandy in the resulted png. With Chrome and Safari however, not so much.

Versions

Chrome 54.0.2840.98 (64-bit)
Safari 10.0 (11602.1.50.0.10)

```
Access to Image at 'https://assets.myapp.com/image.png' from origin 'https://assets.myapp.com' has been blocked by CORS policy: No 'Access-Control-Allow-Origin' header is present on the requested resource. Origin 'https://myapp.com' is therefore not allowed access.

Could not load https://assets.myapp.com/image.png
```

Then, if I alter the source code of `saveSvgAsPng`, or more specifically, change this:

```
img.crossOrigin="anonymous";
        href = href || image.getAttribute('href');
        if (href) {
          img.src = href;
          img.onload = function() {
```
into this ->
```
img.crossOrigin="anonymous";
        href = href || image.getAttribute('href');
        if (href) {
          img.src = href + '?anonymous=true'; // <-- let's add a query parameter to fool cache
          img.onload = function() {
```

and voilá, it works. The images load and I get no errors about CORS violations.

So I assume Chrome and Safari are somehow caching the image and don't bother with the whole crossOrigin-thingie?

I wonder if this is:

a) something I've misconfigured myself
b) bug in Chrome and Safari
c) bug in `saveSvgAsPng`

**Comment by imfycc** [comment]
The same problem

**Comment by imfycc** [comment]
https://bugs.chromium.org/p/chromium/issues/detail?id=409090

**Comment by exupero** [comment]
This should be fixed by #164.
